packages/core/src/api-wrappers/defaultApis.ts

```diff
@@ -78,91 +78,134 @@ export const defaultApis = [
     deps: {
       discoveryApi: discoveryApiRef,
       oauthRequestApi: oauthRequestApiRef,
+      configApi: configApiRef,
     },
-    factory: ({ discoveryApi, oauthRequestApi }) =>
-      GoogleAuth.create({ discoveryApi, oauthRequestApi }),
+    factory: ({ discoveryApi, oauthRequestApi, configApi }) =>
+      GoogleAuth.create({
+        discoveryApi,
+        oauthRequestApi,
+        environment: configApi.getString('auth.environment'),
```

Comment:
The schema says the field is optional, but in the code you do not use `getOptionalString`. This is the cause of the build errors.

Reply:
my bad! corrected them